GPU/Workflow/src/GPUWorkflowSpec.cxx

```diff
@@ -220,6 +220,7 @@ void GPURecoWorkflowSpec::init(InitContext& ic)
 
     // initialize TPC calib objects
     initFunctionTPC();
+    mFastTransformHelper->init(ic);
```

Comment:
can you move this to initFunctionsTPC? Would make sense thematically.

Reply:
ok, but this required to pass `ic` to initFunctionTPC.